stix_shifter/stix_transmission/src/modules/security_advisor/security_advisor_status_connector.py

```diff
@@ -0,0 +1,17 @@
+from ..base.base_status_connector import BaseStatusConnector
+from ..base.base_status_connector import Status
+
+class SecurityAdvisorStatusConnector(BaseStatusConnector):
+    def __init__(self, host, auth):
+        self.host = host
+        self.auth = auth
+
+    def create_status_connection(self, search_id):
+        
+        return_obj = {}
+        return_obj['success'] = True
+        return_obj['status'] = 'COMPLETED'
+        return_obj['progress']  = '100'
+
+        return return_obj
+        
```

Comment:
sorry we probably missed this before. this is a wrong way to create status object. this is send a wrong status to the user if user creates a connector with wrong credentials and host. they wont even know until we send a query to the api. user needs to know the correct status as soon as a datasource connection is created in the UI.
To implement this correct way, you need to make at least a call the the api not necessarily has to be a full query. it can be a POST call to auth endpoint or even a HEAD request to make sure you receive a successful response. For example, you can have a look at some other modules like QRadar, BigFix etc